packages/ui-components/src/__tests__/DeploymentSteps.test.ts

```
@@ -670,6 +679,63 @@ describe('DeploymentSteps', () => {
+		await user.click(deployButton);
+
+		// Wait for the disclaimer modal to be called
+		let onAcceptCallback: () => void;
+		await waitFor(() => {
+			expect(defaultProps.handleDisclaimerModal).toHaveBeenCalled();
+			const callArgs = (defaultProps.handleDisclaimerModal as Mock).mock.calls[0][0];
+			expect(callArgs).toHaveProperty('onAccept');
+			expect(typeof callArgs.onAccept).toBe('function');
+			onAcceptCallback = callArgs.onAccept;
+		});
+		onAcceptCallback!();
```

Biome (1.9.4)

[error] Forbidden non-null assertion.

Unsafe fix: Replace with optional chain operator ?. This operator includes runtime checks, so it is safer than the compile-only non-null assertion operator


(lint/style/noNonNullAssertion)